packages/core/src/components/icon/icon.story.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+import { c, classy } from '../../utils';
+
+/**
+ * `.ods-icon` requires SVG sprite to be included in your app. If you wish to
+ * use individual SVGs, import them directly from `@onfido/castor-icons/svg`
+ * instead.
+ *
+ * https://github.com/onfido/castor-icons
```

Comment:
Tha'ts actually not accurate.

If individual SVGs are used, you won't even be able to use only `.ods-icon`, instead you will have to use mask etc.

I would discourage such as it's an advanced usage.

```suggestion
 * `.ods-icon` requires SVG sprite to be included in your app.
 *
 * https://github.com/onfido/castor-icons
```